app/docs/dev/installation/docker.md

```diff
@@ -34,6 +34,33 @@ This example will run Kuma in `standalone` mode for a "flat" deployment, but the
 **Note**: By default this will run Kuma with a `memory` [store](/docs/{{ page.version }}/documentation/configuration#store), but you can use a persistent storage like PostgreSQL by updating the `conf/kuma-cp.conf` file.
 {% endtip %}
 
+#### 2.1 Authentication (optional)
+
+Running administrative tasks (like generating a dataplane token) require authentication by a token or a connection via localhost.
+
+##### 2.1.1 Localhost
+
+For `kuma-cp` to recognize requests issued to docker published port it needs to run the container in the host network.
+To do this, add `--network="host"` parameter to the `docker run` command from point 2.
+
+##### 2.1.2 Authenticating via token
+
+You can also configure `kumactl` to access `kuma-dp` from the container.
+To do this use the following commands:
+
+```sh
+docker ps # note kuma-cp container id
+
+TOKEN=$(bash -c 'docker exec -it <KUMA_CP_CONTAINER_ID> wget -q -O - http://localhost:5681/global-secrets/admin-user-token' | jq -r .data | base64 -d)
```

Comment:
Nitpick perhaps but putting this all together and using `TOKEN=..` makes it seem like you can copy and paste this but then <KUMA_CP_CONTAINER_ID> has to be set manually. I'd make it either copy-pastable, which might be tough with the container ID, or break it up into multiple steps and use `$KUMA_CP_CONTAINER_ID`